src/amo/pages/Guides/index.js

```diff
@@ -331,15 +347,29 @@ export class GuidesBase extends React.Component<InternalProps> {
   getGuidesSections = (
     sections: Array<SectionsType>,
   ): React.ChildrenArray<React.Node> => {
-    const { addons, i18n } = this.props;
+    const { _log, addons, i18n } = this.props;
+
+    const hasAddonsLoaded =
+      Object.keys(addons).length !== 0 && !this.props.loading;
 
     return sections.map((section) => {
       const linkParts = getLocalizedTextWithLinkParts({
         i18n,
         text: section.exploreMore,
       });
 
-      const addon = addons[section.addonGuid] || null;
+      let addon;
+
+      if (hasAddonsLoaded) {
+        if (addons[section.addonGuid] === null) {
+          addon = null;
+          _log.error(
+            `There is an error with add-on guid: ${section.addonGuid}`,
```

Comment:
Maybe you can write `Could not load add-on with GUID: ${section.addonGuid}`? We know that it is an error thanks to the log level, so we could give a hint ("... load") in the content.

Reply:
I agree that message is better so I've updated this here (to "Could not load.."), but I am not sure what you think I should do regarding the hint:/ 
is this sufficient or are you suggesting I do something else too?